contract_manager/scripts/send_message_to_wormhole.ts

```diff
@@ -0,0 +1,55 @@
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+import { WormholeEmitter, loadHotWallet } from "../src/governance";
+
+const parser = yargs(hideBin(process.argv))
+  .usage(
+    "Usage: $0 --cluster <cluster> --wallet-path <path_to_wallet_file> --message <message>"
+  )
+  .options({
+    cluster: {
+      type: "string",
+      demandOption: true,
+      describe: "The Pyth cluster to use for sending the message",
+    },
+    walletPath: {
+      type: "string",
+      demandOption: true,
+      describe: "Path to the wallet file to use for sending the message",
+    },
+    message: {
+      type: "string",
+      demandOption: true,
+      describe: "The message to send to the wormhole",
```

Comment:
please specify the format (probably hex) and whether we should include the leading `0x`